marklogic-data-hub/src/test/java/com/marklogic/hub/flow/FlowRunWithDataHubOperatorTest.java

```diff
@@ -167,5 +191,9 @@ public void testRunFlow() throws Exception {
         String file = sh.get();
         Assertions.assertNotNull(file);
         Assertions.assertTrue(file.contains("ingest.csv"));
+        //Flow with custom xqy mapping function
+        Assertions.assertTrue(JobStatus.FINISHED.toString().equalsIgnoreCase(resp1.getJobStatus()));
```

Comment:
Side note - should use assertEquals here, with the expected value first. The benefit is that if the assertion fails, JUnit will print the expected and actual values, which is more helpful than just seeing "assertion failed".